data/net/minecraft/world/level/levelgen/synth/SimplexNoise.mapping

```diff
@@ -1,13 +1,20 @@
 CLASS net/minecraft/world/level/levelgen/synth/SimplexNoise
+	COMMENT A generator for a single octave of Simplex noise.
+	FIELD p [I
+		COMMENT A permutation array used in noise generation.
+		COMMENT This is populated with the values [0, 256) and shuffled. Despite the array declared as 512 length, only the first 256 values are used.
+		COMMENT
+		COMMENT @see #p(int)
```

Comment:
```suggestion
		COMMENT @see net.minecraft.world.level.levelgen.synth.SimplexNoise#p(int)
```

Reply:
See prior explanation re FQN within the same class.